include/internal/deprecated.h

```diff
@@ -26,5 +26,8 @@
 # define OPENSSL_SUPPRESS_DEPRECATED
 
 # include <openssl/macros.h>
+# include <stddef.h>
+
+unsigned char *SHA1_int(const unsigned char *d, size_t n, unsigned char *md);
```

Comment:
Dont know that this is the right place for this..

Reply:
Moved to `include/crypto/sha.h`